wrench/src/wrench.rs

```diff
@@ -224,17 +224,6 @@ impl Wrench {
             blob_image_handler: Some(Box::new(blob::CheckerboardRenderer::new(callbacks.clone()))),
             disable_dual_source_blending,
             chase_primitive,
-            #[cfg(feature = "gfx")]
-            heaps_config: webrender::HeapsConfig {
```

Comment:
why don't we keep `heaps_config` as the value (instead of the path) and do the deserialization right here (instead of  the renderer)?

Reply:
Because if the deserialization is in renderer, I don't have to duplicate it for both wrench and gecko.

Reply:
I think that would be better still. It's not an obvious decision to read something from a file, so it's best to push it up the stack.